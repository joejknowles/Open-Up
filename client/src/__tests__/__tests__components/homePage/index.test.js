import Home from '../../../components/homePage';

import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

const shallowRenderer = ReactTestUtils.createRenderer();

it('renders home page as before', () => {
  const notFound = shallowRenderer.render(<Home />);
  expect(notFound).toMatchSnapshot();
});
