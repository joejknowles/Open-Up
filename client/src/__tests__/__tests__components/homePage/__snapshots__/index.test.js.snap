exports[`test renders home page as before 1`] = `
<div>
  <h3>
    Venue List
  </h3>
  <Link
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/venue">
    Venue 1
  </Link>
</div>
`;
